### PR TITLE
social jsx

### PR DESCRIPTION
**Pull request by Recogn1z**
- I already tried my best to match the tailwind width and height with figma, e.g for 238x268 px, I only can get roughly rem in tailwind like w-60 h-64. 
-  it seems icon pictures are not so clear if make it bigger
-  there is no link for facebook and highlander link, so I just put `<a herf="/" target="_blank"></a>`
-  not good at viewing figma design with two items gap ( hope  I can get some help with this problem)
test

### CODE REVIEW

File: src/components/home/Socials.jsx [hunk 1]
```diff
@@ -1,5 +1,84 @@
...
+              <FaFacebookSquare className=" w-24 h-24 fill-rishi-black" />
+            </a>
+            <div className="flex  font-bold   text-2xl w-28 h-8 items-center justify-center text-rishi-black">
+              Facebook
+            </div>
+          </div>
+        </div>
+
+        <div className="border-2 border-rishi-green rounded-2xl flex justify-center items-center w-60  h-64 bg-rishi-gray">
+          <div className="shadow bg-rishi-white rounded-2xl w-56 h-60 flex flex-col justify-around items-center">
```
Comment (pondohoo) [inline src/components/home/Socials.jsx:41]: add py-3 so the text is not so far away from the icon

File: src/components/home/Socials.jsx [hunk 2]
```diff
@@ -1,5 +1,84 @@
...
+              <FaLink className="w-20 h-20 fill-rishi-black" />
+            </a>
+            <div className="flex  font-bold   text-xl w-44 h-9 items-center justify-center text-rishi-black">
+              Highlander Link
+            </div>
+          </div>
+        </div>
+
+        <div className="border-2 border-rishi-orange rounded-2xl flex justify-center items-center w-60  h-64 bg-rishi-gray">
+          <div className="shadow bg-rishi-white rounded-2xl w-56 h-60 flex flex-col justify-around items-center">
```
Comment (pondohoo) [inline src/components/home/Socials.jsx:56]: add py-3 so the text is not so far away from the icon

**Comment by Recogn1z** [comment]
- change 4 <a> to make Link warpping the whole squire.
- delete bg gray 
- add py-3 for 4 squires to make text closer to icon
- add highlander link and facebook link to regarding <Link>

**Comment by Recogn1z** [comment]
second version img:
image

**Review by pondohoo — APPROVED** [review]

**Pull request merged**